js/application.js

```diff
@@ -1,4 +1,5 @@
 // instantiate a StockView
 $('document').ready(function() {
-  var stockView = new StockView();
+	stockView = new StockView();
```

Comment:
this is one workaround to allow the model to rerender the view, but can you think of a way to do this without using a global variable? - I would recommend adding this functionality after you construct your `Stock()` object, using a callback! (because the ajax call is asynchronous.)